dm/syncer/validator_cond.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package syncer
+
+import (
+	"strings"
+)
+
+type Cond struct {
```

Comment:
I'm sorry that I can't understand this naming😂.

What does `Cond` mean? And what is its full name? @D3Hunter

Reply:
condition

Reply:
I see that `Cond` is used in `validateDeletedRows` and `validateInsertAndUpdateRows`.

So, why name it condition? I don't know how to establish their connection?

Reply:
struct `Cond` is used to formulate the `where` clause when querying the downstream database.

Reply:
I see! It means the condition in SQL...

I didn't think of it first. 😅